File: Aplikacija/client-app/src/app/modules/events/models/event.ts
import {TimeSlot} from "../interfaces/time-slot";

export class Event{
  constructor(
    public id: string,
    public name: string,
    public description: string,
    public sport: string,
    public date: string,
    public numberOfParticipants: number,
    public maxParticipants: number,
    public timeSlot: TimeSlot,
    public isCancelled: boolean,
    public hostName: string,
    public participants: {appUserId: string, username: string}[]
  ) {
  }
}

File: Aplikacija/Application/Profiles/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Domain;

namespace Application.Profiles
{
    public class Event
    {
        public Guid Id { get; set; }
        public string Title { get; set; }

        public DateTime Date { get; set; }

        public string Sport { get; set; }

        public string Description { get; set; }

        public AppUser AppUser { get; set; }

        public string HostName { get; set; }

        public int NumOfParticipants { get; set; }

        public int MaxParticipants { get; set; }

        public Domain.TimeSlot TimeSlot { get; set; }

        public ICollection<Profile> Participants { get; set; }
    }
}